backend/open_api_spec.yml

```diff
@@ -24,8 +24,62 @@ paths:
 
   /curation/collections:
     get:
-      description: Get all Collections metadata
+      description: Get all Collections metadata. Include authentication information to retrieve private collections as well. 
```

Comment:
This doesn't square with the `security` mechanism. I thought we were restricting all curator api functionality to curators with access tokens.

Reply:
that required field is only used for json that is in a request, not the response.